{
  "resourceType" : "ValueSet",
  "id" : "2.16.840.1.113883.10.20.5.9.40",
  "meta" : {
    "versionId" : "20",
    "lastUpdated" : "2023-12-21T17:43:03.000-05:00",
    "profile" : [
      "http://hl7.org/fhir/StructureDefinition/shareablevalueset",
      "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/computable-valueset-cqfm",
      "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/publishable-valueset-cqfm"
    ]
  },
  "extension" : [
    {
      "url" : "http://hl7.org/fhir/StructureDefinition/valueset-author",
      "valueContactDetail" : {
        "name" : "CDC NHSN Author"
      }
    },
    {
      "url" : "http://hl7.org/fhir/StructureDefinition/resource-lastReviewDate",
      "valueDate" : "2025-02-21"
    },
    {
      "url" : "http://hl7.org/fhir/StructureDefinition/valueset-effectiveDate",
      "valueDate" : "2021-04-07"
    }
  ],
  "url" : "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.10.20.5.9.40",
  "identifier" : [
    {
      "system" : "urn:ietf:rfc:3986",
      "value" : "urn:oid:2.16.840.1.113883.10.20.5.9.40"
    }
  ],
  "version" : "20210407",
  "name" : "NHSNCOVID19CaseClassification",
  "title" : "NHSN COVID19 Case Classification",
  "status" : "active",
  "experimental" : false,
  "date" : "2021-04-07T01:00:18-04:00",
  "publisher" : "CDC NHSN",
  "jurisdiction" : [
    {
      "coding" : [
        {
          "system" : "urn:iso:std:iso:3166",
          "code" : "US"
        }
      ]
    }
  ],
  "purpose" : "(Clinical Focus: COVID-19 conditions associated with confirmed case status.),(Data Element Scope: Problem Observation code (CDA) or Condition code (FHIR). Used to identify patients with confirmed COVID-19.),(Inclusion Criteria: ),(Exclusion Criteria: )",
  "compose" : {
    "include" : [
      {
        "system" : "http://snomed.info/sct",
        "concept" : [
          {
            "code" : "1240591000000102",
            "display" : "Severe acute respiratory syndrome coronavirus 2 not detected (finding)"
          },
          {
            "code" : "840539006",
            "display" : "Disease caused by Severe acute respiratory syndrome coronavirus 2 (disorder)"
          }
        ]
      }
    ]
  },
  "expansion" : {
    "identifier" : "urn:uuid:694a4027-f7ec-4848-89d9-c1df544022d3",
    "timestamp" : "2025-05-23T21:40:35-04:00",
    "total" : 2,
    "contains" : [
      {
        "system" : "http://snomed.info/sct",
        "version" : "http://snomed.info/sct/731000124108/version/20250301",
        "code" : "1240591000000102",
        "display" : "Severe acute respiratory syndrome coronavirus 2 not detected (finding)"
      },
      {
        "system" : "http://snomed.info/sct",
        "version" : "http://snomed.info/sct/731000124108/version/20250301",
        "code" : "840539006",
        "display" : "Disease caused by severe acute respiratory syndrome coronavirus 2 (disorder)"
      }
    ]
  }
}
